lib/identity/IdentityFactory.js

```diff
@@ -106,7 +101,7 @@ class IdentityFactory {
    * @return {IdentityCreateTransition}
    */
   createIdentityCreateTransition(identity) {
-    const lockedOutPoint = identity.getLockedOutPoint().toString('base64');
+    const lockedOutPoint = identity.getLockedOutPoint().toBuffer();
```

Comment:
Actually, you can pass `identity.getLockedOutPoint()` directly and it should work.